development-vm/replication/common-args.sh

```diff
@@ -25,7 +25,7 @@ OPTIONS:
     -q       Skip MySQL import
     -e       Skip Elasticsearch import
     -t       Skip Mapit import
-    -k       Keep backups after import (will delete by default)
+    -k       Delete backups after import (will keep by default)
```

Comment:
I'm wondering whether we should use a different letter, as most of us may not initially realise this has changed and still include `-k` when we want to keep the backups and then realised after it's deleted them!

Reply:
maybe `del` as `d` is already taken?

Reply:
nice, `d` was gone but `del` is a good idea!

Reply:
sadly, that is parse as `-d el` so sets the directory as `el`

Reply:
Ah! How about `x`?